CommandLine/CommandLine.swift

```diff
@@ -545,18 +545,26 @@ public class CommandLine {
    * on inout protocol function parameters in Xcode 7 beta 1 (rdar://21372694).
    */
 
-  /**
-   * Prints a usage message.
-   * 
-   * - parameter to: An OutputStreamType to write the error message to.
-   */
+    
+    /// If set, will print this out after Usage: name [options]
+    /// Useful to inform user of any command you might support after the options.
+    public var suplementalHelp: String?
+    
+    
+    /**
+     * Prints a usage message.
+     *
+     * - parameter to: An OutputStreamType to write the error message to.
+     */
+    
+    
   #if swift(>=3.0)
     public func printUsage<TargetStream: OutputStream>(_ to: inout TargetStream) {
       /* Nil coalescing operator (??) doesn't work on closures :( */
       let format = formatOutput != nil ? formatOutput! : defaultFormat
 
       let name = _arguments[0]
-      print(format("Usage: \(name) [options]", .About), terminator: "", to: &to)
+      print(format("Usage: \(name) [options]" + self.suplementalHelp ?? "", .About), terminator: "", to: &to)
```

Comment:
That is where the put in the info about the 'command'   so it looks like:   [options] command

I can just check for and add the space with an if statement.

To tell you the truth, this is a bit older [before I changed how commands are programmed].  I might want to more directly tie in to checking for commands.  I also wanted to ask the commands for their documentation.  So I am thinking of removing the suplmentalHelp anyway [I think I put a todo around somewhere talking about this].